src/index.js

```diff
@@ -10,6 +10,8 @@ const PsModuleMap = require('./PsModuleMap');
 const Psc = require('./Psc');
 const PscIde = require('./PscIde');
 const dargs = require('./dargs');
+const execSync = require('child_process').execSync
```

Comment:
All right, my bad: I've forgot to remove `src` query parameter in webpack.config.js, that's why source paths have been ignore. I probably should add a warning about `pscPackage` being ignored if `src` is set.
So looks like there were no synchronization issues, just me.